<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkSliceBasedInterpolatorControls</class>
 <widget class="QWidget" name="QmitkSliceBasedInterpolatorControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>93</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Slice-Based Interpolation</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="m_gbControls">
     <property name="title">
      <string>Slice-Based Interpolation</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="m_btApply">
        <property name="toolTip">
         <string>Accept current preview</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/SegmentationUI.qrc">
          <normaloff>:/Qmitk/AcceptCurrentInterpolation.png</normaloff>:/Qmitk/AcceptCurrentInterpolation.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>34</width>
          <height>34</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_btApplyForAllSlices">
        <property name="toolTip">
         <string>Accept complete preview</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/SegmentationUI.qrc">
          <normaloff>:/Qmitk/AcceptAllInterpolations.png</normaloff>:/Qmitk/AcceptAllInterpolations.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>34</width>
          <height>34</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/SegmentationUI.qrc"/>
 </resources>
 <connections/>
</ui>
